Makefile

```diff
@@ -409,9 +418,9 @@ tools/$(HTML_COMPRESSOR):
 	cd tools; wget --no-check-certificate https://htmlcompressor.googlecode.com/files/$(HTML_COMPRESSOR) -O $(HTML_COMPRESSOR)
 else
 tools/$(HTML_COMPRESSOR):
-	$(Q) mkdir -p tools
-	cd tools; wget https://github.com/yui/yuicompressor/releases/download/v2.4.8/$(YUI_COMPRESSOR)
-	cd tools; wget https://htmlcompressor.googlecode.com/files/$(HTML_COMPRESSOR)
+#	$(Q) mkdir -p tools
+#	cd tools; wget https://github.com/yui/yuicompressor/releases/download/v2.4.8/$(YUI_COMPRESSOR)
+#	cd tools; wget https://htmlcompressor.googlecode.com/files/$(HTML_COMPRESSOR)
```

Comment:
I got a "url not found" during compilation for the htmlcompressor. Seems the path is not correct (anymore) or it was a temporary problem. So I just downloaded both manually and placed them into the tools folder.